### ISSUE TITLE
Quit logs mpv_render_context_render() not being called or stuck

### ISSUE THREAD

**Issue by low-batt** [iina/iina#4831]
**System and IINA version:**

- macOS 1.3.4
- IINA 13.6.3

**Expected behavior:**
 IINA shuts down cleanly.

**Actual behavior:**
During shutdown `mpv` reports  `mpv_render_context_render` is not being called:
```text
[   9.121][d][vd] Uninit decoder.
[   9.138][d][cplayer] Terminating demuxers...
[   9.139][d][cplayer] Done terminating demuxers.
[   9.139][v][cplayer] finished playback, success (reason 2)
[   9.139][i][cplayer] 
[   9.326][v][vo/libmpv] mpv_render_report_swap() not being called.
[   9.531][v][vo/libmpv] mpv_render_context_render() not being called or stuck.
[   9.535][d][libmpv_render] flushing shader cache
[   9.536][d][cplayer] Run command: quit, flags=64, args=[code="0"]
```

**Steps to reproduce:**
Quit IINA while the video is still playing.

- [x] MPV does not have this problem.

**How often does this happen?**
Some of the time.

**Comment by low-batt** [comment]
In [this comment](https://github.com/iina/iina/issues/4822#issuecomment-1961926358) on issue #4822 @krackers explains the VO can only shutdown after it exits the flip loop. That requires the display link to be running. In the comment that follows @krackers points out that in `VideoView.uninit` IINA is shutting down the display link before shutting down the `mpv` render context, thus disrupting shutdown.

**Comment by low-batt** [comment]
This was fixed by the changes in PR #5057.
